rotkehlchen/api/v1/schemas.py

```diff
@@ -1027,72 +1018,111 @@ def make_history_base_entry(self, data: dict[str, Any], **_kwargs: Any) -> dict[
             )
             return {'events': events}
 
-    class CreateEvmSwapEventSchema(BaseSwapEventSchema, BaseEvmEventSchema):
+    class CreateEvmSwapEventSchema(BaseEvmEventSchema):
         sequence_index = fields.Integer(required=True)
+        timestamp = TimestampMSField(required=True)
+        location = LocationField(required=True)
+        event_identifier = fields.String(required=False, load_default=None)
+        spend_amounts = fields.List(AmountField(validate=validate.Range(min=ZERO, min_inclusive=False)), required=True, validate=validate.Length(min=1))  # noqa: E501
+        spend_assets = fields.List(AssetField(expected_type=Asset, form_with_incomplete_data=True), required=True, validate=validate.Length(min=1))  # noqa: E501
+        receive_amounts = fields.List(AmountField(validate=validate.Range(min=ZERO, min_inclusive=False)), required=True, validate=validate.Length(min=1))  # noqa: E501
+        receive_assets = fields.List(AssetField(expected_type=Asset, form_with_incomplete_data=True), required=True, validate=validate.Length(min=1))  # noqa: E501
+        fee_amounts = fields.List(AmountField(load_default=None, validate=validate.Range(min=ZERO, min_inclusive=False)), required=False)  # noqa: E501
+        fee_assets = fields.List(AssetField(load_default=None, expected_type=Asset, form_with_incomplete_data=True), required=False)  # noqa: E501
+        identifiers = fields.Dict(
+            keys=fields.String(validate=validate.OneOf(['spend', 'receive', 'fee'])),
+            values=fields.List(fields.Integer()),
+            required=True,
+        )
+        location_labels = fields.Dict(
```

Comment:
same here for the default loading. Just use `load_default=XXX` instead of using `data.get()` in the implementation of a custom `@post_load` function. Use the library we are working with, instead of doing things manually